### DISCUSSION TITLE
Update necessary

### DISCUSSION THREAD

**Discussion by rogo19** [nextcloud/all-in-one#2734] [Questions] [answered]
Hi

I am running this docker image on a Raspberry Pi with external USB Hdd attached for storage.

Today my Nextcloud docker installation is not ok, I assume after a recent update. I have this installation and upgrade procedure fully automated as per your instructions using nightly Borg backups that will also update the containers when there are available updates.

My startpage says: "Update necessary", and that I have to "update using the command line since automatic updates are turned off in config.php"

I have never had any issues before and am not used to troubleshoot in this context with nextcloud docker image. "It just works" :-). But now there is a problem. How shoud I move forward and troubelshoot this?

I assume it would be a mistake to try and update the Nextcloud installation manually since I use docker?

Best regards,
Roger

**Comment by corwin42** [comment]
Same here.
Nextcloud web interface just shows that an update is necessary and the nextcloud instance is not accessible.

I too had configured automatic updates on the AIO Admin page a few days ago.

What can we do to get Nextcloud up again?

**Comment by mpanhans** [comment]
Fwiw, I just updated Nextcloud AIO from v5.0.0 to v6.0.0 with no issues. I do not have automatic updates though, I manually click the update button in the web interface.

**Comment by KindlyPizza** [comment]
Also experiencing this issue today,

**Comment by rtertiaer** [comment]
I had this problem this morning too. As a quick fix, [following the update instructions & upgrading to NextCloud 27](https://github.com/nextcloud/all-in-one/discussions/2208) brought everything back online for me. Cheers @mpanhans for the tip!

**Comment by arcstur** [comment, marked as answer]
Had the same problem here. I [used this](https://github.com/nextcloud/all-in-one/discussions/1787) and it solved!! Very easy.

**Comment by rogo19** [comment]
"sudo docker exec --user www-data -it nextcloud-aio-nextcloud php occ upgrade" solved this problem for me. Thank you mpanhans, rtasson and arcstur for pointing me in the right direction.
